### ISSUE TITLE
Unable to start espeakup screen reader in Debian environment.

### ISSUE THREAD

**Issue by w-sx** [termux/termux-app#3343]
### Problem description

I installed the Debian environment using proot-distro and hoped to start the espeakup to assist me in obtaining the content on the screen.
However, termux reported an error 'Unable to open the softsynth device: No such file or directory'.
I don't know if this error can be resolved under the termux framework, but I really need espeakup's help, otherwise it will be difficult for me to get information on the screen.

### Steps to reproduce the behavior.

open termux app and run command.
pkg install proot-distro
proot-distro install debian
proot-distro login debian
apt install espeakup
espeakup -d
Run espeakup in debug mode, Error reproduction.

### What is the expected behavior?

espeakup can run normally and read the content on the screen

### System information

* Termux application version:0.118.0
* Android OS version:13
* Device model:Xiaomi 10S
